src/Uno.Extensions.Core.Generators/PropertySelectorGenerator.cs

```diff
@@ -44,4 +43,26 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
 		// We use the Implementation as the generated code does not alter the SemanticModel (only generates a registry).
 		context.RegisterImplementationSourceOutput(provider, (context, souce) => _tool.Generate(context, souce.Left, souce.Right));
 	}
+
+	private static bool IsCandidate(SyntaxNode node)
+	{
+		if (node is InvocationExpressionSyntax invocationExpression)
+		{
+			foreach (var arg in invocationExpression.ArgumentList.Arguments)
+			{
+				if (arg.Expression is SimpleLambdaExpressionSyntax simpleLambda && IsValidLambda(simpleLambda))
+				{
+					return true;
+				}
+			}
+		}
+
+		return false;
+
+		static bool IsValidLambda(SimpleLambdaExpressionSyntax simpleLambda)
+		{
+			return !(simpleLambda.Parameter is null or { IsMissing: true } or { Identifier.ValueText.Length: <= 0 } ||
+				simpleLambda.Body is null or { IsMissing: true });
```

Comment:
Personally I prefer "positive condition" (i.e. no `!` around the whole condition). Also you can use much more pattern matching :)

```suggestion
			return simpleLambda is { Parameter: { IsMissing: false, Identifier.ValueText.Length: > 0}, Body.IsMissing: false };
```